### PR TITLE
add migration for google_cloud_cpp 2.5.0

### PR DESCRIPTION
**Pull request by h-vetinari**
Not sure why the bot doesn't open PRs for this one anymore - we already discussed this in the [PR](https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/3618) for 2.3.0, and the bot didn't open one for either 2.4 or 2.5 🤷 

CC @conda-forge/google-cloud-cpp @hmaarrfk

### CODE REVIEW

**Comment by hmaarrfk** [comment]
This is strange.

**Comment by h-vetinari** [comment]
> This is strange.

Where did you get the 2.5.5. from?

File: recipe/migrations/google_cloud_cpp250.yaml [hunk 1]
```diff
@@ -8,4 +8,7 @@ __migrator:
 google_cloud_cpp:
 - '2.5.0'
 
+libgoogle_cloud:
+- '2.5.5'
```
Comment (h-vetinari) [inline recipe/migrations/google_cloud_cpp250.yaml:12]:
```suggestion
- '2.5.0'
```

Comment (h-vetinari) [reply recipe/migrations/google_cloud_cpp250.yaml:12]: @hmaarrfk 
Did you see this?

Comment (hmaarrfk) [reply recipe/migrations/google_cloud_cpp250.yaml:12]: oops.

**Pull request merged**